fastplotlib/layouts/_plot.py

```diff
@@ -117,7 +132,18 @@ def show(
             self.toolbar = ToolBar(self)
             self.toolbar.maintain_aspect_button.value = maintain_aspect
 
-        return VBox([self.canvas, self.toolbar.widget])
+        if not sidecar:
+            return VBox([self.canvas, self.toolbar.widget])
+
+        if self.sidecar is None:
+            # sidecar doesn't like unpacking when kwargs are `None`
+            if sidecar_kwargs is not None:
+                self.sidecar = Sidecar(**sidecar_kwargs)
+            else:
+                self.sidecar = Sidecar()
+
+        with self.sidecar:
+            return display(VBox([self.canvas, self.toolbar.widget]))
```

Comment:
I'm curious, what happens if you just do `display(...)` without the return?

Reply:
opens a sidecar, does not put plot in sidecar lol but just displays below cell

Reply:
O_O ok lol